### ISSUE TITLE
Update docs to call out `Default` constructor for Style

### ISSUE THREAD

**Issue by epage** [SergioBenitez/yansi#14]
Because `new` was the only constructor listed in the [heading](https://docs.rs/yansi/0.5.0/yansi/struct.Style.html#foreground-color-constructors), I didn't check for a `default` and what it meant.  It was only after digging through the source that I realized that this was an option and it did what I wanted (nothing).

**Comment by SergioBenitez** [comment]
This is documented for Paint and Color, but besides the impl itself, it's not called out. A single line addition in the constructors should do it!

**Comment by SergioBenitez** [comment]
Closing in favor of #23.
